java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -861,6 +863,17 @@ public Table filter(ColumnVector mask) {
     return contiguousSplit(nativeHandle, indices);
   }
 
+  /** Transposes a table
```

Comment:
The code looks fine, but this is a very expensive operation. At a minimum, I would like to have it documented that it can cause a huge explosion in host memory and unless there is really no other option I don't want to support it.